RBS/Install-RSCRubrikBackupService.ps1

```diff
@@ -0,0 +1,173 @@
+<#
+.SYNOPSIS
+Install the Rubrik Backup Service on a remote machine
+
+.DESCRIPTION
+Script will download the Rubrik Backup Service from the RubrikCluster provided. The script will then push the msi, perform a quiet install 
+and then configure the service to start under a specific service account. 
+
+.PARAMETER RubrikCluster
+Represents the IP Address or Name of the Rubrik Cluster
+
+.PARAMETER OutFile
+Download location of the RubrikBackupService.zip from the RubrikCluster
+
+.PARAMETER ComputerName
+Server to install the Rubrik Backup Service On
+
+.EXAMPLE
+.\Install-RubrikBackupService.ps1 -RubrikCluster 172.21.8.51 -computername cl-sql2012-1a
+
+.NOTES
+    Name:               Install Rubrik Backup Service
+    Created:            1/03/2019
+    Author:             Chris Lumnah
+   
+#>
+#requires -modules RubrikSecurityCloud, PSDscResources
+param(
+    # Rubrik Cluster name or ip address
+    [Parameter(Mandatory=$true)]
+    [string]$RubrikCluster,
```

Comment:
RubrikCluster would indicate to me this is a Rubrik cluster object.

Reply:
specifically, I'd rather see this be an actual object input, or name change the parameter to RubrikClusterName.